src/OrchardCore.Modules/OrchardCore.Contents/Startup.cs

```diff
@@ -58,6 +58,8 @@ namespace OrchardCore.Contents;
 
 public sealed class Startup : StartupBase
 {
+    public override int Order { get; } = OrchardCoreConstants.ConfigureOrder.Contents;
```

Comment:
This should not be needed. Make sure that Publish later depends on Content will ensure that publish later is always registered after content

Reply:
Ah interesting.